bootstrap/roles/appliance-build.bootstrap/tasks/main.yml

```diff
@@ -18,6 +18,16 @@
 - shell: git rev-parse --show-toplevel 2>/dev/null
   register: toplevel
 
+- apt:
+    name: aptitude
+    state: present
+
+- apt:
```

Comment:
Is there a reason we use multiple “apt” invocations? I thought they could be pulled into a single one, but until I can look at the docs, idk for sure. 

I’m fine with this either way.

Reply:
I'm going to integrate this now so we can get the builds going again (I just got a passing `git-ab-pre-push` run) but I'll look at the docs and see if it can be simplified.